Makefile

```diff
@@ -0,0 +1,19 @@
+#
```

Comment:
Plesae use oasis instead of introducing yet another Makefile variant into our code.